utils/get_supported_scylla_base_versions.py

```diff
@@ -133,8 +143,7 @@ def get_supported_scylla_base_versions(self, supported_versions) -> list:  # pyl
 
     def filter_rc_only_version(self, base_version_list: list) -> list:
         LOGGER.info("Filtering rc versions from base version list...")
-        if base_version_list and len(base_version_list) > 1:
-            # if there's only release candidates in this repo, skip this version
+        if (base_version_list and len(base_version_list) >= 1) and self.scylla_version not in ('enterprise', 'master'):
```

Comment:
When `if base_version_list` is `True` then `len(base_version_list) >= 1` will also be `True`.
So, the `len(base_version_list) >= 1` part can be removed as redundant.

Reply:
well, technically you are right, but what is for some reason it gets a string, or any other type that is not an integer?
i expect it to be a positive integer (anything bigger than 0)... would `if base_version_list` be enough?

Reply:
anyway, changed to your suggestion